### PR TITLE
docs/auth/jwt: add resolved version in upgrade guide

### PR DESCRIPTION
**Pull request by fairclothjm**

### CODE REVIEW

File: website/content/partials/known-issues/1_16-jwt_auth_config.mdx
Comment (schavis) [inline website/content/partials/known-issues/1_16-jwt_auth_config.mdx:18]:
```suggestion
<Note title="Resolved">
   Resolved in Vault v1.16.2
</Note>
```

Suggest making the resolved note more eye-catching

**Review by schavis — CHANGES_REQUESTED** [review]

**Comment by fairclothjm** [comment]
@schavis Thanks, the requested changes have been resolved.

**Review by mladlow — COMMENTED** [review]
Would you mind making this consistent with how we've added other resolutions? I haven't seen us add a separate note-style block before.

I believe for consistency we'd add to line 10 something like "This issue is addressed in Vault 1.16.2 and later" or something along those lines.

**Comment by fairclothjm** [comment]
Thanks @mladlow I would be happy to make the change. @schavis suggested the note-styling and I would not want to override their suggestion without approval.

**Comment by fairclothjm** [comment]
@mladlow addressed in https://github.com/hashicorp/vault/pull/27015/commits/2bef5b8ef73bd84ac514da7290dcd2b62e14a0bb

**Review by mladlow — APPROVED** [review]
@schavis I've gone ahead and approved this because I think it takes into account your feedback as well, and I want to unblock @fairclothjm from merging.

**Comment by fairclothjm** [comment]
Merging is blocked. I believe I require approval from @schavis since they request changes.

**Comment by fairclothjm** [comment]
@schavis Hello, I am unable to merge this until I get your approval since you requested changes. Thanks!

**Pull request merged**
